components/edit-collective/sections/Tiers.js

```diff
@@ -292,7 +292,8 @@ class Tiers extends React.Component {
         label: intl.formatMessage(this.messages['maxQuantity.label']),
         description: intl.formatMessage(this.messages['maxQuantity.description']),
         when: (tier, collective) =>
-          [TICKET, PRODUCT].includes(tier.type) || (tier.type === TIER && ![FUND, PROJECT].includes(collective.type)),
+          [TICKET, PRODUCT, MEMBERSHIP].includes(tier.type) ||
```

Comment:
Maybe we could extract this array to a `TIER_TYPES_WITH_QUANTITY` variable? It's very likely that someone adding a new tier type here will forget to update the other array you've introduced below

Reply:
Thanks for the idea. I've just done that. 👍🏽